assets/src/scss/pages/post/_article-content.scss

```diff
@@ -100,8 +100,8 @@
   }
 
   p:not(.article-list-item-meta) a,
-  > ul li a,
-  > ol li a {
+  ul li a,
```

Comment:
In Posts lists aren't direct children of the `post-content` div, because they're in an `article` element